src/cfnlint/maintenance.py

```diff
@@ -44,8 +44,9 @@ def update_resource_spec(region, url):
     multiprocessing_logger.debug('Downloading template %s into %s', url, filename)
 
     # Check to see if we already have the latest version, and if so stop
-    if not url_has_newer_version(url):
-        return
+    # commented out for now due to https://github.com/aws-cloudformation/cfn-python-lint/pull/1383#issuecomment-629891506
+    # if not url_has_newer_version(url):
+    #     return
```

Comment:
It would be better to keep the `url_has_newer_version()` call, but instead of returning from the function continue on instead, just skipping the downloads. This would let you do the patching regardless of whether new external specs are available or not.

```python
    # Check to see if we already have the latest version, otherwise download it
    if url_has_newer_version(url):
        spec_content = get_url_content(url, caching=True)
        spec = json.loads(spec_content)
    else:
        with open(filename, 'r') as f:
            spec = json.load(f)
```

Tests will likely need to be adjusted.

Reply:
> It would be better to keep the `url_has_newer_version()` call, but instead of returning from the function continue on instead, just skipping the downloads. This would let you do the patching regardless of whether new external specs are available or not.

Agreed, just using this in the meantime

Reply:
@Tro95 maybe just an undocumented `--force` flag for now that skips this `url_has_newer_version()` check?
```bash
cfn-lint --update-specs --force
```

Reply:
That seems like a good short-term solution

Reply:
[just added a bit more to `--update-specs` beyond the official Resource Specifications here as well](https://github.com/aws-cloudformation/cfn-python-lint/pull/1682)